code/modules/mob/living/say.dm

```diff
@@ -59,6 +59,29 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	"÷" = "cords"
 ))
 
+/mob/living/proc/Ellipsis(original_msg, chance = 50, keep_words)
+    if(chance <= 0)
+        return "..."
+    if(chance >= 100)
+        return original_msg

```

Comment:
I know the code was like this before but this seems opposite what it should be. Below, `chance` increases ellipsis occurrence, but here it seems to decrease it.